ansible_collections/arista/cvp/molecule/cv_change_control_v3/test_show_and_remove_cc.yml

```
@@ -0,0 +1,120 @@
+      arista.cvp.cv_change_control_v3:
+        state: show
+        name: "{{ change.name }}"
+      register: cv_show_cc_by_name
+
+    - name: Check change control
+      vars:
+        query: "[?value.change.name=='{{ change.name }}']"
+      set_fact:
+        show_cc_by_name: "{{ cv_show_cc_by_name.data.change_controls | json_query(query) | first }}"
+
+    - assert:
+        that:
+          - cv_show_cc_by_name.changed == false
+          - show_cc_by_name.value.change.name == "{{ change.name }}"
+          - show_cc_by_name.value.change.notes == "{{ change.notes }}"
+          - show_cc_by_name.value.change.stages.values != {}
+
+    - name: "Show an unknown change control on {{inventory_hostname}} using CC name"
+      arista.cvp.cv_change_control_v3:
+        state: show
+        name: "Unknown-CC-1234567890" # unknown cc name, does not exist on CV
+      register: cv_show_cc_by_name
+
+    - name: Check change control
+      assert:
+        that:
+          - cv_show_cc_by_name.changed == false
+          - cv_show_cc_by_name.data.change_controls == []
+          - cv_show_cc_by_name.failed == false
+
+    - name: "Show an unknown change control on {{inventory_hostname}} using change_id field"
+      arista.cvp.cv_change_control_v3:
+        state: show
+        change_id: [ "1234567890" ]
+      register: cv_show_unknown_id
+
+    - name: Check change control
+      assert:
+        that:
+          - cv_show_unknown_id.changed == false
+          - cv_show_unknown_id.data.change_controls == [None]
+          - cv_show_unknown_id.failed == false
+
+######################################
+##       REMOVE CHANGE CONTROL      ##
+######################################
+
+    - name: "Remove a change control on {{inventory_hostname}} using change_id field"
+      arista.cvp.cv_change_control_v3:
+        state: remove
+        change_id: [ "{{ cv_change_control_test.data.id }}" ]
+      ignore_errors: yes
```

Why do we need to ignore errors here?

---

I guess, its was by mistake. Fixed it !